pkg/flags/unique_urls_test.go

```diff
@@ -89,5 +93,29 @@ func TestNewUniqueURLsWithExceptions(t *testing.T) {
 		if uv.String() != tests[i].rs {
 			t.Fatalf("#%d: expected %q, got %q", i, tests[i].rs, uv.String())
 		}
+		require.Equal(t, tests[i].exp, uv.Values)
+		require.Equal(t, uv.String(), tests[i].rs)
```

Comment:
With the addition of these two lines, you should remove line 89 ~ 95. Or remove these two lines and keep it as it's.